python.md

```diff
@@ -1233,6 +1234,18 @@ Related reading:
 
 - http://stackoverflow.com/questions/19074745/python-docstrings-descriptions-vs-comments
 
+### <a name="spelling">Spelling</a>
+
+Consistent spelling is helpful because it decreases the risk of incorrect variable references. Where applicable, follow the spelling used in external libraries. For example, the Django Rest Framework uses the American English spelling for `serializer` which is therefore prefered. Elsewhere aim for conformity with the pre-existing choices. For example, given the following split:
```

Comment:
It's a persuasive argument, but I'd prefer it in the PR comment rather than actually in the style guide. The whole section could just be simplified to say "Prefer American English for comments and naming.".

I agree with your point about conformity; as the Zen of Python puts it, 'a foolish consistency is the hobgoblin of little minds'.  Personally I think it's all covered by the word "prefer" in the sentence above.

Reply:
> The whole section could just be simplified to say "Prefer American English for comments and naming."

Yep, convention titles should be statements like this. We could do with some conventions for our conventions...